TransformationPlayground/playground/transform_expert/prompting/knowledge/__init__.py

```diff
@@ -0,0 +1,33 @@
+from transform_expert.parameters import SourceVersion, TargetVersion, TransformType, TransformLanguage
+
+from transform_expert.prompting.knowledge.es_6_8 import INDEX_GUIDANCE as es_6_8_index_guidance, INDEX_KNOWLEDGE as es_6_8_index_knowledge
+from transform_expert.prompting.knowledge.os_2_17 import INDEX_GUIDANCE as os_2_17_index_guidance, INDEX_KNOWLEDGE as os_2_17_index_knowledge
+
+
+def get_source_guidance(source_version: SourceVersion, target_version: TargetVersion, input_shape_type: TransformType, transform_language: TransformLanguage) -> str:
```

Comment:
Guidance is things like, "if you have multiple type mappings, prefer to split them into separate indices".  As a result - it's closely tied to the specific source/target versions.  Think of this as a set of default hints we provide the LLM when there's ambiguity on the multiple, valid routes it could take.